### PR TITLE
Feature/responsive refactoring

### PR DESCRIPTION
**Pull request by oleg331**
Rewrite styles into three-level nesting
Remove all zeros before dots

### CODE REVIEW

File: scss/components/blog/_slider-post.scss [hunk 1]
```diff
@@ -1,85 +1,153 @@
...
+    .short-info-post {
+        color: $color-short-info-post;
+        margin-bottom: 8px;
+        line-height: 14px;
+
+        @include clearfix;
+    }
+
+    .info-post-item {
+        float: left;
```
Comment (kobweb) [inline scss/components/blog/_slider-post.scss:63]: It seems like you forgot clearfix for parent?

File: scss/components/blog/_slider-post.scss [hunk 2]
```diff
@@ -1,85 +1,153 @@
...
-                    right: -1px;
-                    width: calc(50% - 17px);
-                    height: 0;
-                }
+        &:not(:last-child) {
+            margin-right: 14px;
+        }
+
+        .icons-blog {
+            display: inline-block;
```
Comment (kobweb) [inline scss/components/blog/_slider-post.scss]: ` float` makes element `display: block`...

File: scss/components/blog/_slider-post.scss (hunk 1 above)
Comment (oleg331) [reply scss/components/blog/_slider-post.scss:63]: Added `@include clearfix`

File: scss/components/blog/_slider-post.scss (hunk 2 above)
Comment (oleg331) [reply scss/components/blog/_slider-post.scss]: Deleted.

**Pull request merged**